fixtures/mock-oauth2-provider/package.json

```diff
@@ -2,10 +2,11 @@
   "name": "@loopback/mock-oauth2-provider",
   "version": "0.0.3",
   "description": "mocks the oauth2 authorization flow",
-  "private": true,
```

Comment:
I thought we were intentionally keeping this package private ?

Reply:
Per #5520, when the `example-passport-login` is cloned with `lb4 example`, `npm i` fails as this package is devDep of `example-passport-login`.